### PR TITLE
Bug 2000379: fix: ensure operator images are included in rendered bundles related images

### PR DESCRIPTION
**Pull request by joelanford**
Signed-off-by: Joe Lanford <joe.lanford@gmail.com>

Upstream-repository: operator-registry
Upstream-commit: f5dcf40f01b8707194fdde07818c8b440ee010fa

### CODE REVIEW

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Comment by timflannagan** [comment]
Related: https://github.com/openshift/origin/pull/26444.

File: staging/operator-registry/internal/action/render.go [hunk 1]
```diff
@@ -322,6 +331,33 @@ func getRelatedImages(b *registry.Bundle) ([]declcfg.RelatedImage, error) {
 	if err = json.Unmarshal(*rawValue, &relatedImages); err != nil {
 		return nil, err
 	}
+
```
Comment (Xia-Zhao-rh) [inline staging/operator-registry/internal/action/render.go:334]: Hi, @joelanford 
I use this branch to test "opm render" a bundle, but still cannot get operator images.
RC:
If there no spec.RelatedImages in csv, this function will return at line 327.

Comment (joelanford) [reply staging/operator-registry/internal/action/render.go:334]: Oh good catch! I'll make another upstream fix for that case.

**Comment by timflannagan** [comment]
Holding so the bot doesn't go crazy. I just moved the 4.10 parent BZ manually to MODIFIED as these changes already exist and we just need QE coverage to make the bot happy again.

/hold

**Pull request merged**